src/renderer/viz/expressions/Animation.js

```diff
@@ -70,16 +73,17 @@ let waitingForOthers = new Set();
  * @class
  * @api
  */
-export class Animation extends BaseExpression {
+
+class ScalarAnimation extends BaseExpression {
     constructor (input, duration = 10, fade = new Fade()) {
         checkMaxArguments(arguments, 3, 'animation');
 
         duration = implicitCast(duration);
         input = implicitCast(input);
         const originalInput = input;
 
-        if (input.isA(Property)) {
-            input = linear(input, globalMin(input), globalMax(input));
+        if (input.isA(Property) || (input.isA(ClusterTime) && input.type === 'timerange')) {
```

Comment:
We should probably create some external & shared consts for this 'timerange', right?